taxonomies/minerals.txt

```diff
@@ -150,6 +150,9 @@ wikipedia:en:https://en.wikipedia.org/wiki/Mineral
 vegan:en:yes
 vegetarian:en:yes
 
+en:ammonium bicarbonate
+es:bicarbonato de amonico,bicarbonato de amonio,bicarbonato amonico,bicarbonato amonio
```

Comment:
We should only add the correct forms:  bicarbonato amonico, and bicarbonato de amonio

"de" is a stopword anyway, so even if there are incorrect forms in the ingredients list, we will recognize them.

Reply:
ok, removed